.ci_support/migrations/cuda12x.yaml

```diff
@@ -67,6 +67,8 @@ __migrator:
       - 11.2                       # [(linux or win64) and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
       - 11.8                       # [(linux or win64) and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
       - 12.0                       # [(linux or win64) and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
+      - 12.4                       # [linux and ppc64le and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
+      - 12.6                       # [((linux and not ppc64le) or win64) and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
```

Comment:
The `ordering` section needed an update for the move to CUDA 12.4 (`ppc64le` only) & 12.6 (all others)

xref: https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/6720

Reply:
Thanks, John!